src/auth/pages/Login/Login.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link as L, withRouter, RouteComponentProps } from 'react-router-dom';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { Form as _Form, FormikActions } from 'formik';
+import { CbAdmins, Response } from '../../../api';
+import LoginForm, { FormValues } from './LoginForm';
+import { H1, H4 } from '../../../components/Headings';
+
+
+/**
+ * Types
+ */
+interface LoginProps extends RouteComponentProps {}
+
+
+/**
+ * Styles
+ */
+const Link = styled(L)`
+  margin-top: 2rem;
+`;
+
+
+/**
+ * Helpers
+ */
+
+// Submit handler creator
+const createSubmitHandler = (props: LoginProps) =>
+  (values: FormValues, actions: FormikActions<FormValues>) =>
+    CbAdmins.login(values)
+      .then(() => props.history.push('/'))
+      .catch((error) => {
+        const res = error.response;
+
+        if (Response.statusEquals(res, 400)) {
```

Comment:
Do we want an error handler like the visitor app?

Reply:
We already have `redirectOnError`, but we need to handle these three cases separately anyway. I can use it in the final case.